### ISSUE TITLE
Create Patal TV page

### ISSUE THREAD

**Issue by ariefrahmansyah** [palembang-digital/website#32]
- Create new page to show Patal YouTube videos
- Add new menu to navbar (Patal TV)

**Comment by taufik230301** [comment]
the video src are imported from utube link?

**Comment by taufik230301** [comment]
![2020-09-13](https://user-images.githubusercontent.com/51141717/93017294-a2793f80-f5f1-11ea-9266-408b9480f9f8.png)

**Comment by ariefrahmansyah** [comment]
> the video src are imported from utube link?

Yes. We will have a backend API for this, but for now, you can use a placeholder for the UI.

**Comment by taufik230301** [comment]
okay, now im still exploring about image and video gallery in reactjs, that makes beautiful UI for Patal TV

**Comment by taufik230301** [comment]
I got some problem after I've tried to build this Patal TV
![2020-09-19](https://user-images.githubusercontent.com/51141717/93665796-9c78d800-faa3-11ea-82f7-777e9b3ffbf6.png)

**Comment by taufik230301** [comment]
I've explored. and I think here is the problem, 
![2020-09-19 (1)](https://user-images.githubusercontent.com/51141717/93665851-0abd9a80-faa4-11ea-94a1-3d056429d1bf.png)
